src/posts/2020-10-28-dyslexia-fonts.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Dyslexia fonts
+description: A quick overview of dyslexia fonts.
+category: Assistive technology
+author: Hamsa Harcourt
+date: 2020-10-28
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - assistivetechnology
+---
+
+According to Oxford Advanced Learner's Dictionary:
+
+> Dyslexia is a slight disorder of the brain that causes difficulty in reading and spelling, for example, but does not affect intelligence
+
+Dyslexia affects up to 1 in 5 people. However, dyslexia like most conditions, ranges from a mild disability to an extremely disabling one. Since people with dyslexia transpose characters, fonts are an important facet to consider.
+
+
+## Case study
+
+When designing fonts, it's common practice to flip and rotate a character to form other characters. Letters b, p, d and q are usually created using this method. However, this causes problems for people with dyslexia. This is because it makes the letters easy to swap. As a result, people with dyslexia prefer fonts where all the characters are unique. 
+
+
+## Introducing dyslexia fonts
+
+Dyslexia fonts are typefaces designed specially for people with dyslexia. They are designed so that each character is unique and distinguishable. Dyslexia fonts make each character unique by using thicker lines in parts of the letters. Also, the letters are slanted a bit.  Some dyslexia fonts include: Dyslexie, Open dyslexic, Gill dyslexic, Read Regular etc.
+
+![an example of dyslexia fonts](/img/posts/2020-10-28-dyslexia-fonts/dyslexia-font.png)
+ 
+However, according to recent studies, it has been proven that dyslexia fonts do **not** help dyslexics **read faster and make fewer mistakes**. In fact, most dyslexics prefer Verdana and Helvetica to  other fonts , including dyslexia fonts.
```

Comment:
This is a huge dissertation and potentially a generalization. We need to link to these findings cited in a credible source or sources.